src/server/worldserver/Master.cpp

```diff
@@ -111,9 +111,9 @@ class FreezeDetectorRunnable : public ACORE::Runnable
 void Master::LodaModuleConfigSettings()
```

Comment:
Maybe LoadModuleConfigSettings ?

Reply:
@Winfidonarleyan  yes. I didn't notice it.:expressionless: